packages/advanced-logic/src/extensions/payment-network/erc20/fee-proxy-contract.ts

```diff
@@ -23,6 +23,8 @@ export default class Erc20FeeProxyPaymentNetwork<
       'celo',
       'alfajores',
       'fuse',
+      'bsc',
```

Comment:
only `bsctest` is supported as the deployment on `bsc` have not been done.

Reply:
I've replaced bsc by bsctest for this first test.